flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/model/ModelUdf.java

```diff
@@ -0,0 +1,132 @@
+package org.apache.flink.cdc.runtime.operators.model;
+
+import org.apache.flink.cdc.common.data.ArrayData;
+import org.apache.flink.cdc.common.data.GenericArrayData;
+import org.apache.flink.cdc.common.types.DataType;
+import org.apache.flink.cdc.common.types.DataTypes;
+import org.apache.flink.cdc.common.udf.UserDefinedFunction;
+
+import com.google.gson.Gson;
+import dev.langchain4j.data.document.Metadata;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.openai.OpenAiEmbeddingModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ModelUdf implements UserDefinedFunction {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ModelUdf.class);
+    private static final String DEFAULT_API_KEY =
+            "sk-WegHEuogRpIyRSwaF5Ce6fE3E62e459dA61eFaF6CcF8C79b";
+    private static final String DEFAULT_MODEL_NAME = "text-embedding-ada-002";
+    private static final int DEFAULT_TIMEOUT_SECONDS = 30;
+    private static final String DEFAULT_BASE_URL = "https://api.gpt.ge/v1/";
```

Comment:
Passing these "default" API keys and endpoints aren't meaningful. Request tokens might ran out, endpoints might expire.